knowledge-base/html5-viewer-styles-broken-with-kendo-2022-1-119.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Some Html5 Report Viewer Styles are Broken with Kendo Themes 2022.1.119
+description: How to fix the styles of the Html5 Report Viewer when using Kendo Themes 2022.1.119
+type: how-to
+page_title: Kendo Themes 2022.1.119 Break the Html5 Report Viewer Styles
+slug: html5-viewer-styles-broken-with-kendo-2022-1-119
+position: 
+tags: 
+ticketid: 1551073
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td></td>
```

Comment:
The version is missing.

Reply:
It is relevant to all Reporting versions that try to use this specific Kendo version. That's why the Reporting version is not included.